templates/frontends-pipeline-pr-checks/Jenkinsfile

```diff
@@ -36,17 +33,20 @@ pipeline {
     stages {
         stage('Clone Git Repository') {
             agent { label 'insights' }
-            steps {
-                script {
-                    sh '''
-                        env
-                    '''
-                }
+            environment {
+                IMG_TAG=sh(script: "git rev-parse --short=8 HEAD", returnStdout: true).trim()
```

Comment:
I wonder at what time is this IMG_TAG processed and with which repo - like - which `git` command is using, and with which project - the pipelien one ? no steps have run yet AFAIK. 

Can we ensure this IMG_TAG has the value we expect ?

Reply:
Fixed this to be the correct git hash.

Reply:
ah ! this ^ OK :)